src/gdb/gdb.c

```diff
@@ -279,7 +279,7 @@ while (1) {
 				mem_to_hex(sregs + addr, remcom_out_buffer, 4);
 			} else if (addr >= 0x300 && addr < 0x400) {
 				strcpy((char *)remcom_out_buffer,
-						"out of scope");
```

Comment:
"out of scope" is a more meaningful message if its correct ?

Reply:
Yes it is, however the error message must contain only HEX characters 0..F otherwise it may cause GDB to close the session.